### ISSUE TITLE
Wrong URL not found toast

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#1318]
**Summary:** 

URL not found toast is visible when license is null (I am not sure if this case is possible). The related code is here:
```
if (licenseLink(media) != null) {
            license.setOnClickListener(v -> openWebBrowser(licenseLink(media)));
        } else {
            Toast toast = Toast.makeText(getContext(), getString(R.string.null_url), Toast.LENGTH_SHORT);
            toast.show();
        }
```
However I can go to the license page by clicking license, so it shouldn't be null. 

**Steps to reproduce:** 

I faced this issue with some images. Just click on images to see media details page. However, this doesn't happen on each images, so try several:) 

**Add System logs:**

No logs

**Expected behavior:** 

Code should enable if statement instead of else

**Observed behavior:** 

Else statement is runned

**Device and Android version:** 
19 API level, prodDebug

**Screen-shots:** 
https://commons.wikimedia.org/wiki/File:Special_bread_from_Turkey.jpg

![image](https://user-images.githubusercontent.com/3127881/37513212-77e456d4-2915-11e8-86ff-a70d1b7b15b9.png)

**Would you like to work on the issue?**

No:D

**Comment by yashshah7197** [comment]
Hey @neslihanturan I would like to work on this issue. Thanks! 😄

**Comment by knight-shade** [comment]
@neslihanturan I upload a sample image with similar licence `cc-zero`, but it opened in browser.

**Comment by neslihanturan** [comment]
Actually @knight-shade, it is not directly related with license type, I have other images with this license and this toast isn't displayed. However, going media detail view of some pictures (I don't know what is special with them) causes this toast become visible. Even with those pictures we are still able to click on license and go to the correct page. So toast gets displayed while license is not null. 

Since there is not any solution except toast (it is not informative besides being wrong alert) and just removing can solve, I am in favor to find the root cause if we can.

**Comment by knight-shade** [comment]
@neslihanturan That's really strange :confused: That means both the `if` and `else` are getting executed :astonished: , moreover the mentioned toast is the only code that uses string `Error: URL not found`.  I tried with more images, not able to reproduce the bug. Removing only seems to be optimal solution. Shall I create PR for it?

**Comment by neslihanturan** [comment]
Since it is low priority, maybe we can wait someone else to discover this issue with its reason. What so you say @knight-shade ?

**Comment by knight-shade** [comment]
Ya , sure @neslihanturan  :+1:

**Comment by Helvijs-t2t** [comment]
Hi, I just 'joined' this project. I would like to take a look at this as it is labelled "Beginner Friendly". 
Could you please give me some more information on where can I see the full source code for this?

**Comment by knight-shade** [comment]
@Helvijs-t2t In  `MediaDetailFragment` at line `282`.

**Comment by Helvijs-t2t** [comment]
@knight-shade  for me (I am not sure why) it shows it at line `297` in the same file.  However, I am not able to debug anything because of
![capture](https://user-images.githubusercontent.com/32101902/37988020-b7d7817c-31f7-11e8-83bf-a85556fd2a7d.PNG)
![capture1](https://user-images.githubusercontent.com/32101902/37988031-bac1f43a-31f7-11e8-9288-adad41e0e092.PNG)

Any ideas of what have I done wrong that the directory/file called `R` is missing?

**Comment by neslihanturan** [comment]
I don't think R is missing. Android studio sometimes wrong alerts about it. Did you able to run other projects with you Anrdoid Studio setup? If so firstly fork our project, then git clone your fork to your local computer, then open the project with Android Studio. Everything should work properly with this scenario. Make sure your Android Studio is updated, help -> check for updates

**Comment by Helvijs-t2t** [comment]
My mistake on updates. This was because I Updated everything to latest versions and was running `'com.android.tools.build:gradle:3.1.0'`  when this project runs on `'com.android.tools.build:gradle:3.0.1'` after changing parameters this error is not active anymore.

**Comment by neslihanturan** [comment]
There are no extra information except I wrote here @Helvijs-t2t . Thanks for your interest, I will surely write here if I recognize something else. It you would like to contribute in time being, you can check other  beginner friendly tasks:)

**Comment by Helvijs-t2t** [comment]
Could you please share your email with me for a quick question, you know my email address already. Thanks @neslihanturan

**Comment by neslihanturan** [comment]
tur.neslihan@gmail.com

**Comment by sivaraam** [comment]
Has this been resolved in 30d7b5d3?

**Comment by neslihanturan** [comment]
Thanks @sivaraam , yes it should be solved by then.
